manual/vcredist2005/update.ps1

```diff
@@ -0,0 +1,51 @@
+Import-Module AU
+. "$PSScriptRoot\..\..\scripts\Set-DescriptionFromReadme.ps1"
```

Comment:
Since the update makes use of AU, this line is not necessary. Please remove the sourcing of `Set-DescriptionFromReadme.ps1`